<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionForm</class>
 <widget class="QMainWindow" name="ConnectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ConnectoinForm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="Lamp_vLayout"/>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>7</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Адрес сервера:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ServerNameLEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Порт:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="PortLEdit">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
        <property name="inputMask">
         <string>99999</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>7</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="DisconnectButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Разъединить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ConnectButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Сединить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
